windows/QMK Toolbox/MainWindow.cs

```diff
@@ -234,13 +234,15 @@ private void LoadKeyboardList()
                     if (json != null) {
                         var keyboards = JsonConvert.DeserializeObject<List<string>>(json);
                         keyboardBox.Items.Clear();
+                        keyboardBox.Items.Add("--- Select a keyboard to download ---");

```

Comment:
```suggestion
                        keyboardBox.Items.Add("Select a keyboard to download");
```
Just thinking out loud, we might need to subclass this combo box to implement a proper "placeholder" type thing, since apparently WinForms doesn't have that...

Reply:
Found a way to do a placeholder without making a subclass.